interactive-etl/setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -u 
+set -e
+set -o errexit
+
+NAMESPACE=${1:-flink}
+
+printf "\n\nInstalling example components into namespace: %s\n\n" "$NAMESPACE"
+
+# Install CertManager - this is needed by the Flink Kubernetes Operator
+printf "\nChecking for CertManager install\n"
+if kubectl get namespace cert-manager ; then
+    printf "CertManager is already installed\n"
+else
+    kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+fi
```

Comment:
I was trying to standardise on using kustomize in my branch for idempotent application and also separating out the installation of the operators to better fit with the Red Hat builds vs community builds.